crates/base-db/src/input.rs

```diff
@@ -277,6 +277,7 @@ pub struct CrateData {
     pub root_file_id: FileId,
     pub edition: Edition,
     pub version: Option<String>,
+    pub package_id: Option<String>,
```

Comment:
CrateData is agnostic to cargo packages so we shouldn't store this here (and we shouldn't have to either way)